src/UILayer/Window.h

```diff
@@ -33,10 +38,15 @@ class Window : public QMainWindow
    QScopedPointer<QPushButton> connectButton_;
    QScopedPointer<QLineEdit> comPortLineEdit_;
    QScopedPointer<QLabel> connectionStatusLabel_;
-   QScopedPointer<QPushButton> sendKeyDriverControlButton_;
-   QScopedPointer<QPushButton> sendFaultsButton_;
-   QScopedPointer<QPushButton> sendBatteryDataButton_;
-   QScopedPointer<QPushButton> sendCmuDataButton_;
-   QScopedPointer<QPushButton> sendMpptDataButton_;
+   QScopedPointer<QPushButton> sendKeyMotorButton_;
```

Comment:
I know it was me that used the QScopedPointers before but for UI objects like push buttons it is better to have them as a raw pointer. The reason for this is that the QT framework will manage the memory when it is added to it. Having two things try and manage the memory can result in memory corruption (This is why the desktop telem was crashing on exit for a long time).